plugins/fixed-edges/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@blockly/fixed-edges",
+  "version": "0.0.0",
```

Comment:
I think this can be updated to version 1.0.0 since it does not monkey patch core.

Reply:
Updated